src/editor.js

```diff
@@ -760,6 +764,15 @@ export class OdooEditor extends EventTarget {
             this._toRollback === UNBREAKABLE_ROLLBACK_CODE ? false : this._toRollback;
         this._checkStepUnbreakable = false;
     }
+    preventHistoryStep() {
```

Comment:
I would call it `historySkipSteps` and `historyUnskipSteps` to be consistant with the other naming style (historyXxx rather than xxxHistoryXxx) and better represent the functionality of the method.

Reply:
Yep you're right with `skip`. So my suggestion becomes: `historyPauseStep`.

Reply:
`preventHistoryStep` is fine by me.
( I hate `historyUnskipSteps` personaly )